integration_tests/cosmoscli.py

```diff
@@ -1129,6 +1129,21 @@ def gov_propose_token_mapping_change(self, proposal, **kwargs):
             )
         )
 
+    def submit_gov_proposal(self, proposal, **kwargs):
```

Comment:
Maybe we could rename [existing](https://github.com/crypto-org-chain/cronos/blob/f3e063a9cd41db9451879ea802acf5a45db1f8de/integration_tests/cosmoscli.py#L1117) one as `submit_gov_proposal`

Reply:
Yup, `gov_propose_token_mapping_change` has been removed with other refactoring accordingly.